p2p/ANSWERS.md

```diff
@@ -19,9 +19,23 @@ Note that in the literature ("Eclipse Attacks on Bitcoin’s Peer-to-Peer Networ
 Q: How does "diffusion" message spreading work and why is it ineffective against de-anonymization?  
 A: Diffusion: Instead of using fixed intervals to broadcast and relay INV messages, node now uses a random delay timer. So it picks a neighbor at random AND use a random delay timer per peer, until the tx has been broadcasted / relayed to all reachable peers. Diffusion is still ineffective against deanonymization because despite the random timers, the spreading pattern is still structured/symmetric which betrays the originating source (with good probability). See code [here](https://github.com/bitcoin/bitcoin/commit/5400ef6bcb9d243b2b21697775aa6491115420f3).
 
-Q: Why must transaction unlocking scripts only push numbers to be relayed? What output scripts are 'IsStandard'?  
-A: There's a bunch of logic for checking if a transaction is standard, such as number of inputs, if its under the dust threshold, etc. I linked the code for the function on the right (policy.cpp).
-Second link is the definition of what sorts of transaction types are considered standard. Learn more [here](https://github.com/bitcoin/bitcoin/blob/master/src/script/standard.h#L59) and [here](https://github.com/bitcoin/bitcoin/blob/master/src/policy/policy.cpp#L80).
+Q: Why must transaction unlocking scripts only push numbers to be relayed?  
+A: Transaction locking scripts specify the requirements that must be fulfilled to spend a UTXO, and locking scripts generally include placeholders for values that must be provided in the unlocking script. When spending an input, values must be placed into the unlocking script of a transaction, such that any bitcoin node can evaluate the script and determine whether the transaction is authorized to spend the input. The combination of the locking script and the unlocking script in a transaction must result in the “stack” evaluating to a non-zero value for the input to be considered spendable. The only types of values that can exist on the “stack” to be evaluated in transaction execution are numbers. These numbers could represent cryptographic keys, time, or arbitrary values.
+
+Q: What output scripts are 'IsStandard'?  
+A: Bitcoin contains standardness checks for a transaction's version, its inputs, and its outputs. Specific to transaction outputs, the standardness rules are:
+
+- The output must [not be a “dust” (IE spam)](https://github.com/bitcoin/bitcoin/blob/fa0074e2d82928016a43ca408717154a1c70a4db/src/policy/policy.cpp#L126) amount
```

Comment:
nit: can you replace these curly ” with straight quotes instead? I'd prefer not to introduce any curlies to the file.

Reply:
nice catch – corrected via a fixup